lib/membrane_rtsp/rtsp.ex

```diff
@@ -120,6 +137,22 @@ defmodule Membrane.RTSP do
 
   @type headers :: [{binary(), binary()}]
 
+  @spec get_transport(t()) :: any()
+  def get_transport(session) do
+    GenServer.call(session, :get_transport)
+  end
+
+  @spec get_parameter_no_reply(t(), headers(), binary()) :: :ok
+  def get_parameter_no_reply(session, headers \\ [], body \\ ""),
+    do: cast_request(session, "GET_PARAMETER", headers, body)
+
+  @spec play_no_reply(t(), headers()) :: :ok
+  def play_no_reply(session, headers \\ []), do: cast_request(session, "PLAY", headers, "")
```

Comment:
I'm not a fan of these functions suffixed by `_no_reply` and the `cast_request` function. I wanted to generalize this by allowing to pass an option to these request functions (`play`, `get_parameter`), but the default arguments make this difficult. Do you have any ideas how to make this nicer? I thought maybe making `headers`, `body`, `path` and `no_reply` (or something similar) options would make this clearer.